Lombiq.HelpfulExtensions/Manifest.cs

```diff
@@ -101,3 +101,13 @@
         "OrchardCore.Markdown",
     }
 )]
+
+[assembly: Feature(
+    Id = OrchardRecipeMigration,
+    Name = "Lombiq Helpful Extensions - Orchard 1 Recipe Migration",
+    Category = "Development",
+    Description =
+        "Convert Orchard 1's export XML files into Orchard Core recipes. This feature contains the basics like " +
+        "CommonPart and BodyPart, but can be extended with additional converters that only have to handle more " +
```

Comment:
Please mention everything here, so list part too.

Reply:
This description was referring to the parts in the export.xml, while ListPart is an OC thing.  
If I had to include everything this would be pretty long:
- CommonPart
- AutoroutePart
- BodyPart
- TitlePart
- IdentityPart
- ListPart
- GraphMetadata
- If a custom converter fills in the OrchardIds.Parent on the content item, then it also initializes the parent content item's ListPart.

I'd rather detail this info in the readme and mention the readme in the feature description, as this should be a relatively short string.

Reply:
Okay, I agree!